src/client/contexts/ActionButtonContext.ts

```diff
@@ -1,8 +1,8 @@
-import { createContext } from 'react';
+import { createContext, ReactNode } from 'react';
 
 export const ActionButtonContext = createContext<{
-	ActionButton?: JSX.Element;
-	update?: (n?: JSX.Element) => void;
+	ActionButton?: ReactNode;
+	update?: (n?: ReactNode) => void;
```

Comment:
Yeah I think `ReactNode` type is better than `JSX.Element` type. Or maybe `ReactElement`?

Reply:
I'm not opposed to `ReactElement`, but definitely like it over `JSX.Element` because the former allows plain strings and the latter does not.
`ReactNode` allows you to pass an array of components or a fragment.

ReactNode allows us to put multiple buttons using the flex spacing of the existing div, whereas ReactElement requires us to wrap multiple buttons. I really don't think there's a ton of benefit either way.